lab_6_pipeline/pipeline.py

```diff
@@ -19,20 +34,48 @@ def __init__(self, path_to_raw_txt_data: Path):
         Initializes CorpusManager
         """
 
+        self._path_to_raw_txt_data = path_to_raw_txt_data
+        self._storage = {}
+        self._validate_dataset()
+
     def _validate_dataset(self) -> None:
         """
         Validates folder with assets
         """
+        if not self._path_to_raw_txt_data.exists():
+            raise FileNotFoundError
+        if not self._path_to_raw_txt_data.is_dir():
+            raise NotADirectoryError
+
+        meta_files = [meta for meta in self._path_to_raw_txt_data.glob(r'*_meta.json')]
```

Comment:
For the mark 4, meta files are not expected, so it is better to remove working with them so as to let the tests pass

Reply:
changed the score